src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -382,7 +385,7 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
                 }
             }
 
-            const averageOverallScore = (this.averageNumberOfOverallPoints / this.maxNumberOfOverallPoints) * 100;
+            const averageOverallScore = round((this.averageNumberOfOverallPoints / this.maxNumberOfOverallPoints) * 100);
```

Comment:
I would add at least one decimal place here as well.

Reply:
Done in 86b66a17e3722f0f0c70d375ea39e9a39f681e37